pages/blog/index.js

```diff
@@ -49,12 +49,12 @@ export default function BlogIndexPage() {
         </div>
         <div className="relative max-w-7xl mx-auto">
           <div className="text-center">
-            <h2 className="text-3xl leading-9 tracking-tight font-extrabold text-gray-900 sm:text-4xl sm:leading-10">
+            <h2 className="text-3xl my-4 leading-9 tracking-tight font-extrabold text-gray-900 sm:text-4xl sm:leading-10">
               Welcome to our blog!
             </h2>
-            <p className="mt-3 max-w-2xl mx-auto text-xl leading-7 text-gray-500 sm:mt-4">
+            {/* <p className="mt-3 max-w-2xl mx-auto text-xl leading-7 text-gray-500 sm:mt-4">
```

Comment:
I think @iamVP7 meant to remove the period at the end, not the whole sentence :thinking: 

I'd not remove the whole sentence, just the period.

Reply:
Oh I'm sorry, just a minute.

Reply:
No problem, no need to rush :)

Reply:
Am I supposed to raise a new PR? Just pushed the corrected file to forked repo.

Reply:
No, as [you can see here](https://github.com/asyncapi/website/pull/484/files) your change is already there automatically :)

Github compares your forked repo/branch with this repo/master, and the result is this PR. If you change your branch, the PR changes.

Reply:
Yes just saw that. Thanks